### PR TITLE
Add release notes for v17.10.0

### PR DESCRIPTION
**Pull request by GaneshSPatil**
screen shot 2017-09-12 at 11 19 41 am

### CODE REVIEW

File: source/partials/release_notes/_release-17-10-0.md.erb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+**Important:** On Sep 11th, we made an announcement asking users to hold off upgrading to 17.9.0 because of issue (<%= link_to("#3834", "https://github.com/gocd/gocd/issues/#{3834}") %>). This release provides fix for the same. While the issue affects a small number of users, we think it still warrants a new release.
+
+<h4>Critical Bug Fixes</h4>
+* <%= link_to_commit '05598d88fd4dabdde1184faa4fbffc5f9406d0dc', 'Unable to start the server after 17.9.0 upgrade', 3834%>
+
+<h4>Other Changes</h4>
+* <%= link_to_commit '176ebc26438784f50bbdc013ebc704a3a6cd6c8d', 'No plugins visible after 17.9.0 upgrade', 3829%>
```
Comment (maheshp) [inline source/partials/release_notes/_release-17-10-0.md.erb]: `Fix for no plugins visible after 17.9.0 upgrade`

File: source/partials/release_notes/_release-17-10-0.md.erb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+**Important:** On Sep 11th, we made an announcement asking users to hold off upgrading to 17.9.0 because of issue (<%= link_to("#3834", "https://github.com/gocd/gocd/issues/#{3834}") %>). This release provides fix for the same. While the issue affects a small number of users, we think it still warrants a new release.
```
Comment (nyuday) [inline source/partials/release_notes/_release-17-10-0.md.erb]: **Important:** After the 17.9 release, we made an announcement asking users to hold off upgrading their GoCD instance. We found an issue (<%= link_to("#3834", "https://github.com/gocd/gocd/issues/#{3834}") %>) affecting a small set of users. This release provides a fix for the same.

**Comment by ketan** [comment]
There's a couple of docker changes that @varshavaradarajan should be able to help with

File: source/partials/release_notes/_release-17-10-0.md.erb [hunk 3]
```diff
@@ -0,0 +1,42 @@
+**Important:** After the 17.9 release, we made an announcement asking users to hold off upgrading their GoCD instance. We found an issue (<%= link_to("#3834", "https://github.com/gocd/gocd/issues/#{3834}") %>) affecting a small set of users. This release provides a fix for the same.
+
+<h4>Critical Bug Fixes</h4>
+* <%= link_to_commit '05598d88fd4dabdde1184faa4fbffc5f9406d0dc', 'Unable to start the server after 17.9.0 upgrade', 3834%>
+
+<h4>Other Changes</h4>
+* <%= link_to_commit '176ebc26438784f50bbdc013ebc704a3a6cd6c8d', 'Fix for no plugins visible after 17.9.0 upgrade', 3829%>
+
+<h4>Docker Images</h4>
+* Allow users to specify `uid` and `gid` as build args while building a customize docker image from GoCD image (both server and agent).
```
Comment (varshavaradarajan) [inline source/partials/release_notes/_release-17-10-0.md.erb]: building a custom docker image

File: source/partials/release_notes/_release-17-10-0.md.erb [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+* Allow users to specify a volume mount instruction in docker run command.
```
Comment (varshavaradarajan) [inline source/partials/release_notes/_release-17-10-0.md.erb]: I think users were always allowed to volume mount. We just removed the declaration from our dockerfile. Don't know how you'd phrase that. :)

Comment (nyuday) [reply source/partials/release_notes/_release-17-10-0.md.erb]: Does this help? "Made the marking of volume mounts explicit while starting Docker containers"

File: source/partials/release_notes/_release-17-10-0.md.erb [hunk 5]
```diff
@@ -0,0 +1,42 @@
+**Important:** After the 17.9 release, we made an announcement asking users to hold off upgrading their GoCD instance. We found an issue (<%= link_to("#3834", "https://github.com/gocd/gocd/issues/#{3834}") %>) affecting a small set of users. This release provides a fix for the same.
+
+<h4>Critical Bug Fixes</h4>
+* <%= link_to_commit '05598d88fd4dabdde1184faa4fbffc5f9406d0dc', 'Unable to start the server after 17.9.0 upgrade', 3834%>
+
+<h4>Other Changes</h4>
+* <%= link_to_commit '176ebc26438784f50bbdc013ebc704a3a6cd6c8d', 'Fix for no plugins visible after 17.9.0 upgrade', 3829%>
+
+<h4>Docker Images</h4>
```
Comment (varshavaradarajan) [inline source/partials/release_notes/_release-17-10-0.md.erb]: Link to changelog of docker gocd server and docker gocd agent

File: source/partials/release_notes/_release-17-10-0.md.erb [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+* Allow users to specify `uid` and `gid` as build args while building a customize docker image from GoCD image (both server and agent).
+* Allow users to specify a volume mount instruction in docker run command.
+
+<h4>Contributors</h4>
```
Comment (varshavaradarajan) [inline source/partials/release_notes/_release-17-10-0.md.erb]: Add external contributors who have contributed to docker-gocd-server and docker-gocd-agent.

File: source/partials/release_notes/_release-17-10-0.md.erb (hunk 4 above)
Comment (varshavaradarajan) [reply source/partials/release_notes/_release-17-10-0.md.erb]: The users were always allowed.

Comment (varshavaradarajan) [reply source/partials/release_notes/_release-17-10-0.md.erb]: Sorry, didn't read your message. How about we simply remove this change and link to changelog for a list of all changes?

Comment (nyuday) [reply source/partials/release_notes/_release-17-10-0.md.erb]: That works for me. How about? 

<h4> Docker Image</h4>
We have made a few changes to the GoCD Server(link:https://github.com/gocd/docker-gocd-agent/blob/master/CHANGELOG.md) and the GoCD Agent(link: https://github.com/gocd/docker-gocd-agent/blob/master/CHANGELOG.md) Docker images.

**Pull request merged**